src/components/LmsConfigurations/SuccessFactorsConfigForm.jsx

```diff
@@ -0,0 +1,374 @@
+            >
+              <label htmlFor="userType">SAP Success Factors User Type</label>
+              <Input
+                type="select"
+                id="userType"
+                name="userType"
+                defaultValue={config ? config.userType : null}
+                options={[
+                  { value: 'admin', label: 'Admin' },
+                  { value: 'user', label: 'User' },
+                  { value: null, label: 'blank', hidden: true },
+                ]}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-6">
+            <ValidationFormGroup
+              for="transmitTotalHours"
+              helpText="Include totalHours in the transmitted completion data"
+            >
+              <label htmlFor="transmitTotalHours">Transmit Total Hours?</label>
+              <Input
+                type="checkbox"
+                id="transmitTotalHours"
+                name="transmitTotalHours"
+                className="ml-3"
+                checked={transmitTotalHours}
+                onChange={() => this.setState(prevState => ({ transmitTotalHours: !prevState.transmitTotalHours }))}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-4">
+            <ValidationFormGroup
+              for="additionalLocales"
+              helpText="A comma separated list of any additional locales used in SAP (such as 'Dutch' or 'English Canadian'). See SAP's documentation for more examples."
+            >
+              <label htmlFor="additionalLocales">Additional Locales</label>
+              <Input
+                type="text"
+                id="additionalLocales"
+                name="additionalLocales"
+                defaultValue={config ? config.additionalLocales : null}
+              />
+            </ValidationFormGroup>
+          </div>
+        </div>
+
+        <div className="row">
+          <div className="col col-2">
+            <StatefulButton
+              state={submitState}
+              type="submit"
+              id="submitButton"
+              labels={{
+                default: 'Submit',
+                pending: 'Saving...',
+                complete: 'Complete',
+                error: 'Error',
+              }}
+              icons={{
+                default: <Icon className="fa fa-download" />,
+                pending: <Icon className="fa fa-spinner fa-spin" />,
+                complete: <Icon className="fa fa-check" />,
+                error: <Icon className="fa fa-times" />,
+              }}
+              disabledStates={[SUBMIT_STATES.PENDING]}
+              variant="primary"
+              className="ml-3 col"
+            />
+          </div>
+        </div>
+        <div className="row">
+          <div className="col col-3 mt-3">
+            {errorAlert}
```

Comment:
A more "React-y" way to do this would be to have this line be
```
{error && (<div className="form-group is-invalid align-items-left">
          <StatusAlert
            alertType="danger"
            iconClassName="fa fa-times-circle"
            title="Unable to submit config form:"
            message={error}
          />
        </div>)
        }
        ```

Reply:
(But formatted correctly)